analysis_scripts/analysis_utils.R

```diff
@@ -2,6 +2,12 @@ library(jsonlite)
 library(data.table)
 library(dplyr)
 
+
+proj_dir = here()
+source(paste(proj_dir, "/utils/scraper_processing_utils.R", sep=""))
```

Comment:
Same comment as before regarding `file.path`

Reply:
now updated in latest commit: 3fb8701